WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt

```diff
@@ -1113,7 +1127,10 @@ d_juhuria = {
 	culture = juhuro
 	
 	allow = {
-		always = no
+ 		OR = {
+			culture = juhuro
+			religion = tzadikite
+		}
```

Comment:
I disagree. Why shouldn't that culture and religion restore its ancestral duchy?